tests/Storage/CookieTest.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+namespace PhpAb\Storage;
+
+use PHPUnit_Framework_TestCase;
+use phpmock\MockBuilder;
+use phpmock\Mock;
+use phpmock\functions\FixedValueFunction;
+
+/**
+ * During the execution of some of these tests,
+ * global functions might be overwritten
+ */
+class CookieTest extends PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Reset global cookies array and disable 
+     * global function mocks
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+        $_COOKIE = [];
+        \phpmock\Mock::disableAll();
```

Comment:
Move to tearDown. If we do not do at tearDown the last test executed here will not disable it and possibly affect other tests in the same namespace